### ISSUE TITLE
Top bar icon and pause button enlarges after suspending ubuntu 20.04

### ISSUE THREAD

**Issue by arshadmdvr** [gnome-pomodoro/gnome-pomodoro#561]

**Comment by mayconcancini** [comment]
Same behavior in Fedora 34.

**Comment by arandomliz** [comment]
I think correlated, but if this happens one cannot disable the extension anymore as the toggle is grayed out.

**Comment by alcarraz** [comment]
same on ubuntu 21.04 😢  it also prevent screen lock and disables all extensions

**Comment by rimatech987** [comment]
The same in Fedora 34.
It happens every time you return from sleep mode and lock mode.
The icon is enlarged, and ALL extensions are completely locked. You need to disable/enable all extensions to make them work.

**Comment by szpak** [comment]
I didn't observe the mentioned behavior with the [pre-release RPM](https://github.com/gnome-pomodoro/gnome-pomodoro/issues/554#issuecomment-907083485) for 0.19.2 made by Mat on Fedora 34. You might give it a try or wait for the new packages for [just releasted 0.19.2](https://github.com/gnome-pomodoro/gnome-pomodoro/releases/tag/0.19.2).

**Comment by rimatech987** [comment]
> I didn't observe the mentioned behavior with the [pre-release RPM](https://github.com/gnome-pomodoro/gnome-pomodoro/issues/554#issuecomment-907083485) for 0.19.2 made by Mat on Fedora 34. You might give it a try or wait for the new packages for [just releasted 0.19.2](https://github.com/gnome-pomodoro/gnome-pomodoro/releases/tag/0.19.2).

Hi @szpak , thanks for your contribution.
I have tested the 0.19.2 release and it is working perfectly.

**Comment by logrusx** [comment]
I've observed that too, back then I used a desktop computer and it used to happen when the screensaver turned off the display and the break finished. After the screen wake up, the icon was enlarged and the extension didn't work correctly. However I haven't observed such a behavior after migrating from ayatana indicators. I think for the current version 0.20 and judging from the comments above - 0.19.2, this is no longer relevant and this issue may be closed.
